pkg/webhook/validate.go

```diff
@@ -35,12 +35,12 @@ func validateSriovOperatorConfig(cr *sriovnetworkv1.SriovOperatorConfig, operati
 	log.Log.V(2).Info("validateSriovOperatorConfig", "object", cr)
 	var warnings []string
 
-	if cr.GetName() != consts.DefaultConfigName {
-		return false, warnings, fmt.Errorf("only default SriovOperatorConfig is used")
+	if operation == v1.Delete {
```

Comment:
I would like to get more feedback here: from one side, it should be OK to re-create default policy, from other, I would like to allow only update once we've got default policy to not introduce possible race conditions and unnecessary reconcile loops.

I'll press `Request changes` button just to get more attention to this from other maintainers, feel free to ignore my vote